build/configs/imxrt1050-evk/nxp_demo/defconfig

```diff
@@ -293,12 +296,23 @@ CONFIG_IMXRT_FLASH_PART_TYPE="none,smartfs,"
 CONFIG_IMXRT_FLASH_PART_NAME="kernel,imxrt,"
 CONFIG_IMXRT_AUTOMOUNT=y
 CONFIG_IMXRT_AUTOMOUNT_USERFS=y
-CONFIG_IMXRT_AUTOMOUNT_USERFS_DEVNAME="/dev/smart0p8"
+CONFIG_IMXRT_AUTOMOUNT_USERFS_DEVNAME="/dev/smart0p2"
```

Comment:
Please change this to /dev/smart0p1

Reply:
When changing /dev/smart0p1, booting sequence wasn't normally completed.

I think that you need to check it in your side after changing the value.

Reply:
I saw that booting is completed with /dev/smart0p1 even mount is failed. please check again